### ISSUE TITLE
FeedMe Import Error

### ISSUE THREAD

**Issue by jeffaglenn** [verbb/vizy#185]
### Describe the bug

When importing using FeedMe I'm getting an error for entries with Vizy fields that are empty: `Trying to access array offset on value of type null - Vizy.php: 59`. Entries that have content for Vizy fields seem to import fine.

### Steps to reproduce

1. Create CSV or XML to import to entries with a Vizy field
2. Have some Vizy fields blank & others not.
3. Import throws errors for entries with Vizy field content

### Craft CMS version

4.3.1

### Plugin version

2.0.9

### Multi-site?

No

### Additional context

Update: I just entered "test" into the blank fields and things imported fine.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-4 as 2.0.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.10](https://github.com/verbb/vizy/releases/tag/2.0.10)
